src/header/styles.scss

```diff
@@ -75,7 +75,7 @@
 
   &.refresh {
     // Can't use justify-content: center because it won't align with configurable dashboard fixed handle icon
-    row-gap: awsui.$space-static-xxs;
+    row-gap: awsui.$space-xxs;
```

Comment:
`$space-scaled-xxs` ?

Reply:
No, it's supposed to be non-scaling. While making the changes I noticed that we don't really use the `-static` name of the tokens very much.